Big_Integer/main.cpp

```diff
@@ -0,0 +1,476 @@
+#include<bits/stdc++.h>
+
+
+using namespace std;
+
+
+class Integer {
+    int sign;
+    string value;
+
+public:
+    Integer() {
+        //Default Empty Constructor::Dont use delegation...
+    }
+    //positive - negative value separator
+    Integer(string passed) {
+        if (passed[0] == '0') {
+            sign = 0;
+            value = "0";
+        } else if (passed[0] == '-'){
+            sign = -1;
+            value = passed.substr(1); // Copy substring after pos 1
+        } else {
+            sign = +1;
+            value = passed;//if positive store as it's
+        }
+    }
+    Integer(string passed, int Sign) {
```

Comment:
Codacy found an issue: [Function parameter 'passed' should be passed by const reference.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6290669)